ckb-bin/src/subcommand/list_hashes.rs

```diff
@@ -137,17 +144,38 @@ pub fn list_hashes(root_dir: PathBuf, matches: &ArgMatches) -> Result<(), ExitCo
         specs.push((spec_name, spec_hashes));
     }
 
-    println!();
     let length = specs.len();
+
+    // In bundled mode jsons must be trunked in an array, push these brackets manually.
+    if matches.is_present(cli::ARG_BUNDLED) && output_format == "json" {
```

Comment:
Can you show how the JSON looks like?

The TOML format is a map which using spec name as the key. The JSON format should be the same by converting the TOML to JSON.

To simplify the code, I would recommend removing the comments `# Spec: {name}` in TOML, so we can just put all specs into an ordered map and serialize it into the target format.

Reply:
Why ordered map?

To keep the generated file determined.

Reply:
> Can you show how the JSON looks like?
> 

The JSON is now output as an array, not a map. I'll paste it below.

> The JSON format should be the same by converting the TOML to JSON

And if we convert output TOML to JSON, the json will be a map like 
```json
{
  "ckb": {
  // ...
  },
  "ckb_dev": {
  // ...
  }
}
```
which is not same as the output right now.

And i agree with you, this change currently will make future formats intergration fuzzy & verbose.
I'll simplify the code to make it easy to intergrate other formats.


>  I would recommend removing the comments # Spec: {name} in TOML

*But* removing TOML comments will cause diff with `doc/hashes.toml`,  I just want to ensure if this is acceptable, since you said in this comment https://github.com/nervosnetwork/ckb/pull/3564#discussion_r961212510.


## output of `ckb list-hashes -f json`
```json
[{
  "ckb": {
    "spec_hash": "0x3fe86931c2cf5dfa6d7b9447561b5e547937fe4699787bb7ae906218b3f1e6c5",
    "genesis": "0x92b197aa1fba0f63633922c61c92375c9c074a93e85963554f5499fe1450d0e5",
    "cellbase": "0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c",
    "system_cells": [
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_sighash_all)",
        "tx_hash": "0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c",
        "index": 1,
        "data_hash": "0x709f3fda12f561cfacf92273c57a98fede188a3f1a59b1f888d113f9cce08649",
        "type_hash": "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
      },
      {
        "path": "Bundled(specs/cells/dao)",
        "tx_hash": "0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c",
        "index": 2,
        "data_hash": "0x32064a14ce10d95d4b7343054cc19d73b25b16ae61a6c681011ca781a60c7923",
        "type_hash": "0x82d76d1b75fe2fd9a27dfbaa65a039221a380d76c926f378d3f81cf3e7e13f2e"
      },
      {
        "path": "Bundled(specs/cells/secp256k1_data)",
        "tx_hash": "0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c",
        "index": 3,
        "data_hash": "0x9799bee251b975b82c45a02154ce28cec89c5853ecc14d12b7b8cccfc19e0af4",
        "type_hash": null
      },
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_multisig_all)",
        "tx_hash": "0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c",
        "index": 4,
        "data_hash": "0x43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb",
        "type_hash": "0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8"
      }
    ],
    "dep_groups": [
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
        ],
        "tx_hash": "0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c",
        "index": 0
      },
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_multisig_all)"
        ],
        "tx_hash": "0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c",
        "index": 1
      }
    ]
  }
},{
  "ckb_testnet": {
    "spec_hash": "0xdde420d56c10a06749e79701b152ed1f532f4bdf0696b5896dfd97fbc821e231",
    "genesis": "0x10639e0895502b5688a6be8cf69460d76541bfa4821629d86d62ba0aae3f9606",
    "cellbase": "0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f",
    "system_cells": [
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_sighash_all)",
        "tx_hash": "0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f",
        "index": 1,
        "data_hash": "0x709f3fda12f561cfacf92273c57a98fede188a3f1a59b1f888d113f9cce08649",
        "type_hash": "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
      },
      {
        "path": "Bundled(specs/cells/dao)",
        "tx_hash": "0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f",
        "index": 2,
        "data_hash": "0x32064a14ce10d95d4b7343054cc19d73b25b16ae61a6c681011ca781a60c7923",
        "type_hash": "0x82d76d1b75fe2fd9a27dfbaa65a039221a380d76c926f378d3f81cf3e7e13f2e"
      },
      {
        "path": "Bundled(specs/cells/secp256k1_data)",
        "tx_hash": "0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f",
        "index": 3,
        "data_hash": "0x9799bee251b975b82c45a02154ce28cec89c5853ecc14d12b7b8cccfc19e0af4",
        "type_hash": null
      },
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_multisig_all)",
        "tx_hash": "0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f",
        "index": 4,
        "data_hash": "0x43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb",
        "type_hash": "0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8"
      }
    ],
    "dep_groups": [
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
        ],
        "tx_hash": "0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37",
        "index": 0
      },
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_multisig_all)"
        ],
        "tx_hash": "0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37",
        "index": 1
      }
    ]
  }
},{
  "ckb_staging": {
    "spec_hash": "0x1e501d0048dbfee37707432fb5ec4c5e2b0404d887ebc882564c8761ef54e68a",
    "genesis": "0xbc081e6b2e31149c1dc39007f161ed0a0b63d5d30b3b771acc6a3b622133fcc0",
    "cellbase": "0x7295631c414e50a8d9bb73d9845231ac212d10404045c96de7f149ec874ac6b7",
    "system_cells": [
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_sighash_all)",
        "tx_hash": "0x7295631c414e50a8d9bb73d9845231ac212d10404045c96de7f149ec874ac6b7",
        "index": 1,
        "data_hash": "0x709f3fda12f561cfacf92273c57a98fede188a3f1a59b1f888d113f9cce08649",
        "type_hash": "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
      },
      {
        "path": "Bundled(specs/cells/dao)",
        "tx_hash": "0x7295631c414e50a8d9bb73d9845231ac212d10404045c96de7f149ec874ac6b7",
        "index": 2,
        "data_hash": "0x32064a14ce10d95d4b7343054cc19d73b25b16ae61a6c681011ca781a60c7923",
        "type_hash": "0x82d76d1b75fe2fd9a27dfbaa65a039221a380d76c926f378d3f81cf3e7e13f2e"
      },
      {
        "path": "Bundled(specs/cells/secp256k1_data)",
        "tx_hash": "0x7295631c414e50a8d9bb73d9845231ac212d10404045c96de7f149ec874ac6b7",
        "index": 3,
        "data_hash": "0x9799bee251b975b82c45a02154ce28cec89c5853ecc14d12b7b8cccfc19e0af4",
        "type_hash": null
      },
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_multisig_all)",
        "tx_hash": "0x7295631c414e50a8d9bb73d9845231ac212d10404045c96de7f149ec874ac6b7",
        "index": 4,
        "data_hash": "0x43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb",
        "type_hash": "0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8"
      }
    ],
    "dep_groups": [
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
        ],
        "tx_hash": "0xd5780747735fd22c9ba7363bde8afe59061658caa836962867253b03cbda264e",
        "index": 0
      },
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_multisig_all)"
        ],
        "tx_hash": "0xd5780747735fd22c9ba7363bde8afe59061658caa836962867253b03cbda264e",
        "index": 1
      }
    ]
  }
},{
  "ckb_dev": {
    "spec_hash": "0x307bfa518f3dd6ffe6d82317c5ce7eb738db4623402edcc06b5adbee2e3a6b29",
    "genesis": "0x823b2ff5785b12da8b1363cac9a5cbe566d8b715a4311441b119c39a0367488c",
    "cellbase": "0xa563884b3686078ec7e7677a5f86449b15cf2693f3c1241766c6996f206cc541",
    "system_cells": [
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_sighash_all)",
        "tx_hash": "0xa563884b3686078ec7e7677a5f86449b15cf2693f3c1241766c6996f206cc541",
        "index": 1,
        "data_hash": "0x709f3fda12f561cfacf92273c57a98fede188a3f1a59b1f888d113f9cce08649",
        "type_hash": "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
      },
      {
        "path": "Bundled(specs/cells/dao)",
        "tx_hash": "0xa563884b3686078ec7e7677a5f86449b15cf2693f3c1241766c6996f206cc541",
        "index": 2,
        "data_hash": "0x32064a14ce10d95d4b7343054cc19d73b25b16ae61a6c681011ca781a60c7923",
        "type_hash": "0x82d76d1b75fe2fd9a27dfbaa65a039221a380d76c926f378d3f81cf3e7e13f2e"
      },
      {
        "path": "Bundled(specs/cells/secp256k1_data)",
        "tx_hash": "0xa563884b3686078ec7e7677a5f86449b15cf2693f3c1241766c6996f206cc541",
        "index": 3,
        "data_hash": "0x9799bee251b975b82c45a02154ce28cec89c5853ecc14d12b7b8cccfc19e0af4",
        "type_hash": null
      },
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_multisig_all)",
        "tx_hash": "0xa563884b3686078ec7e7677a5f86449b15cf2693f3c1241766c6996f206cc541",
        "index": 4,
        "data_hash": "0x43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb",
        "type_hash": "0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8"
      }
    ],
    "dep_groups": [
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
        ],
        "tx_hash": "0xace5ea83c478bb866edf122ff862085789158f5cbff155b7bb5f13058555b708",
        "index": 0
      },
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_multisig_all)"
        ],
        "tx_hash": "0xace5ea83c478bb866edf122ff862085789158f5cbff155b7bb5f13058555b708",
        "index": 1
      }
    ]
  }
}]
```

Reply:
I've commited these changes.

The reason for not use orderd map: 
- The original output order is `mainnet -> testnet  -> staging -> dev`, if orderd, will cause a huge diff in `docs/hashes.toml`
- The order was defined in [resource/src/template.rs](https://github.com/nervosnetwork/ckb/blob/develop/resource/src/template.rs#L4), the order itself also matches the env in production, so I switched to `LinkedHashMap`

Reply:
And now the JSON output is same with the value converted from TOML, preview as in below:

```json
{
  "ckb": {
    "spec_hash": "0x3fe86931c2cf5dfa6d7b9447561b5e547937fe4699787bb7ae906218b3f1e6c5",
    "genesis": "0x92b197aa1fba0f63633922c61c92375c9c074a93e85963554f5499fe1450d0e5",
    "cellbase": "0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c",
    "system_cells": [
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_sighash_all)",
        "tx_hash": "0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c",
        "index": 1,
        "data_hash": "0x709f3fda12f561cfacf92273c57a98fede188a3f1a59b1f888d113f9cce08649",
        "type_hash": "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
      },
      {
        "path": "Bundled(specs/cells/dao)",
        "tx_hash": "0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c",
        "index": 2,
        "data_hash": "0x32064a14ce10d95d4b7343054cc19d73b25b16ae61a6c681011ca781a60c7923",
        "type_hash": "0x82d76d1b75fe2fd9a27dfbaa65a039221a380d76c926f378d3f81cf3e7e13f2e"
      },
      {
        "path": "Bundled(specs/cells/secp256k1_data)",
        "tx_hash": "0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c",
        "index": 3,
        "data_hash": "0x9799bee251b975b82c45a02154ce28cec89c5853ecc14d12b7b8cccfc19e0af4",
        "type_hash": null
      },
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_multisig_all)",
        "tx_hash": "0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c",
        "index": 4,
        "data_hash": "0x43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb",
        "type_hash": "0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8"
      }
    ],
    "dep_groups": [
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
        ],
        "tx_hash": "0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c",
        "index": 0
      },
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_multisig_all)"
        ],
        "tx_hash": "0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c",
        "index": 1
      }
    ]
  },
  "ckb_testnet": {
    "spec_hash": "0xdde420d56c10a06749e79701b152ed1f532f4bdf0696b5896dfd97fbc821e231",
    "genesis": "0x10639e0895502b5688a6be8cf69460d76541bfa4821629d86d62ba0aae3f9606",
    "cellbase": "0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f",
    "system_cells": [
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_sighash_all)",
        "tx_hash": "0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f",
        "index": 1,
        "data_hash": "0x709f3fda12f561cfacf92273c57a98fede188a3f1a59b1f888d113f9cce08649",
        "type_hash": "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
      },
      {
        "path": "Bundled(specs/cells/dao)",
        "tx_hash": "0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f",
        "index": 2,
        "data_hash": "0x32064a14ce10d95d4b7343054cc19d73b25b16ae61a6c681011ca781a60c7923",
        "type_hash": "0x82d76d1b75fe2fd9a27dfbaa65a039221a380d76c926f378d3f81cf3e7e13f2e"
      },
      {
        "path": "Bundled(specs/cells/secp256k1_data)",
        "tx_hash": "0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f",
        "index": 3,
        "data_hash": "0x9799bee251b975b82c45a02154ce28cec89c5853ecc14d12b7b8cccfc19e0af4",
        "type_hash": null
      },
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_multisig_all)",
        "tx_hash": "0x8f8c79eb6671709633fe6a46de93c0fedc9c1b8a6527a18d3983879542635c9f",
        "index": 4,
        "data_hash": "0x43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb",
        "type_hash": "0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8"
      }
    ],
    "dep_groups": [
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
        ],
        "tx_hash": "0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37",
        "index": 0
      },
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_multisig_all)"
        ],
        "tx_hash": "0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37",
        "index": 1
      }
    ]
  },
  "ckb_staging": {
    "spec_hash": "0x1e501d0048dbfee37707432fb5ec4c5e2b0404d887ebc882564c8761ef54e68a",
    "genesis": "0xbc081e6b2e31149c1dc39007f161ed0a0b63d5d30b3b771acc6a3b622133fcc0",
    "cellbase": "0x7295631c414e50a8d9bb73d9845231ac212d10404045c96de7f149ec874ac6b7",
    "system_cells": [
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_sighash_all)",
        "tx_hash": "0x7295631c414e50a8d9bb73d9845231ac212d10404045c96de7f149ec874ac6b7",
        "index": 1,
        "data_hash": "0x709f3fda12f561cfacf92273c57a98fede188a3f1a59b1f888d113f9cce08649",
        "type_hash": "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
      },
      {
        "path": "Bundled(specs/cells/dao)",
        "tx_hash": "0x7295631c414e50a8d9bb73d9845231ac212d10404045c96de7f149ec874ac6b7",
        "index": 2,
        "data_hash": "0x32064a14ce10d95d4b7343054cc19d73b25b16ae61a6c681011ca781a60c7923",
        "type_hash": "0x82d76d1b75fe2fd9a27dfbaa65a039221a380d76c926f378d3f81cf3e7e13f2e"
      },
      {
        "path": "Bundled(specs/cells/secp256k1_data)",
        "tx_hash": "0x7295631c414e50a8d9bb73d9845231ac212d10404045c96de7f149ec874ac6b7",
        "index": 3,
        "data_hash": "0x9799bee251b975b82c45a02154ce28cec89c5853ecc14d12b7b8cccfc19e0af4",
        "type_hash": null
      },
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_multisig_all)",
        "tx_hash": "0x7295631c414e50a8d9bb73d9845231ac212d10404045c96de7f149ec874ac6b7",
        "index": 4,
        "data_hash": "0x43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb",
        "type_hash": "0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8"
      }
    ],
    "dep_groups": [
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
        ],
        "tx_hash": "0xd5780747735fd22c9ba7363bde8afe59061658caa836962867253b03cbda264e",
        "index": 0
      },
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_multisig_all)"
        ],
        "tx_hash": "0xd5780747735fd22c9ba7363bde8afe59061658caa836962867253b03cbda264e",
        "index": 1
      }
    ]
  },
  "ckb_dev": {
    "spec_hash": "0x307bfa518f3dd6ffe6d82317c5ce7eb738db4623402edcc06b5adbee2e3a6b29",
    "genesis": "0x823b2ff5785b12da8b1363cac9a5cbe566d8b715a4311441b119c39a0367488c",
    "cellbase": "0xa563884b3686078ec7e7677a5f86449b15cf2693f3c1241766c6996f206cc541",
    "system_cells": [
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_sighash_all)",
        "tx_hash": "0xa563884b3686078ec7e7677a5f86449b15cf2693f3c1241766c6996f206cc541",
        "index": 1,
        "data_hash": "0x709f3fda12f561cfacf92273c57a98fede188a3f1a59b1f888d113f9cce08649",
        "type_hash": "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
      },
      {
        "path": "Bundled(specs/cells/dao)",
        "tx_hash": "0xa563884b3686078ec7e7677a5f86449b15cf2693f3c1241766c6996f206cc541",
        "index": 2,
        "data_hash": "0x32064a14ce10d95d4b7343054cc19d73b25b16ae61a6c681011ca781a60c7923",
        "type_hash": "0x82d76d1b75fe2fd9a27dfbaa65a039221a380d76c926f378d3f81cf3e7e13f2e"
      },
      {
        "path": "Bundled(specs/cells/secp256k1_data)",
        "tx_hash": "0xa563884b3686078ec7e7677a5f86449b15cf2693f3c1241766c6996f206cc541",
        "index": 3,
        "data_hash": "0x9799bee251b975b82c45a02154ce28cec89c5853ecc14d12b7b8cccfc19e0af4",
        "type_hash": null
      },
      {
        "path": "Bundled(specs/cells/secp256k1_blake160_multisig_all)",
        "tx_hash": "0xa563884b3686078ec7e7677a5f86449b15cf2693f3c1241766c6996f206cc541",
        "index": 4,
        "data_hash": "0x43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb",
        "type_hash": "0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8"
      }
    ],
    "dep_groups": [
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
        ],
        "tx_hash": "0xace5ea83c478bb866edf122ff862085789158f5cbff155b7bb5f13058555b708",
        "index": 0
      },
      {
        "included_cells": [
          "Bundled(specs/cells/secp256k1_data)",
          "Bundled(specs/cells/secp256k1_blake160_multisig_all)"
        ],
        "tx_hash": "0xace5ea83c478bb866edf122ff862085789158f5cbff155b7bb5f13058555b708",
        "index": 1
      }
    ]
  }
}
```

Reply:
It's acceptable to make the code simple.